getting some weird import python error right now

```
ERROR: /usr/local/google/home/senlu/work/src/k8s.io/kubernetes/cluster/images/kubemark/BUILD:7:1: no such package '@debian//image': Pull command failed: Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/google/home/senlu/.cache/bazel/_bazel_senlu/8bed9ea5a35c4e84a8a5913ca5672618/external/puller/file/puller.par/__main__.py", line 93, in <module>
  File "/usr/local/google/home/senlu/.cache/bazel/_bazel_senlu/8bed9ea5a35c4e84a8a5913ca5672618/external/puller/file/puller.par/__main__.py", line 86, in main
  File "/usr/local/google/home/senlu/.cache/bazel/_bazel_senlu/8bed9ea5a35c4e84a8a5913ca5672618/external/puller/file/puller.par/containerregistry/client/v2_2/v2_compat_.py", line 106, in __init__
  File "/usr/local/google/home/senlu/.cache/bazel/_bazel_senlu/8bed9ea5a35c4e84a8a5913ca5672618/external/puller/file/puller.par/containerregistry/client/v2_2/v2_compat_.py", line 110, in _ProcessImage
  File "/usr/local/google/home/senlu/.cache/bazel/_bazel_senlu/8bed9ea5a35c4e84a8a5913ca5672618/external/puller/file/puller.par/containerregistry/client/v2/docker_image_.py", line 177, in manifest
  File "/usr/local/google/home/senlu/.cache/bazel/_bazel_senlu/8bed9ea5a35c4e84a8a5913ca5672618/external/puller/file/puller.par/containerregistry/client/v2/docker_digest_.py", line 32, in SignedManifestToSHA256
  File "/usr/local/google/home/senlu/.cache/bazel/_bazel_senlu/8bed9ea5a35c4e84a8a5913ca5672618/external/puller/file/puller.par/containerregistry/client/v2/util_.py", line 96, in DetachSignatures
KeyError: 'signatures'
 (/usr/bin/python /usr/local/google/home/senlu/.cache/bazel/_bazel_senlu/8bed9ea5a35c4e84a8a5913ca5672618/external/puller/file/puller.par --directory /usr/local/google/home/senlu/.cache/bazel/_bazel_senlu/8bed9ea5a35c4e84a8a5913ca5672618/external/debian/image --name index.docker.io/library/debian@sha256:f51cf81db2de8b5e9585300f655549812cdb27c56f8bfb992b8b706378cd517d) and referenced by '//cluster/images/kubemark:image'
ERROR: Analysis of target '//cluster/images/kubemark:image' failed; build aborted
INFO: Elapsed time: 4.790s
FAILED: Build did NOT complete successfully (7 packages loaded)
ERROR: Build failed. Not running target
```

--- a/cluster/BUILD
+++ b/cluster/BUILD
@@ -19,6 +19,7 @@ filegroup(
         "//cluster/images/etcd/attachlease:all-srcs",
         "//cluster/images/etcd/rollback:all-srcs",
         "//cluster/images/hyperkube:all-srcs",
+        "//cluster/images/kubemark:all-srcs",
         "//cluster/lib:all-srcs",
         "//cluster/saltbase:all-srcs",
     ],

--- a/cluster/images/kubemark/BUILD
+++ b/cluster/images/kubemark/BUILD
@@ -1,0 +1,33 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build", "docker_push")
+
+docker_build(
+    name = "image",
+    base = "@official_busybox//image",
+    entrypoint = ["/kubemark"],
+    files = ["//cmd/kubemark"],
+)
+
+docker_push(
+    name = "push",
+    image = ":image",
+    registry = "gcr.io",
+    repository = "$(PROJECT)/kubemark",
+    stamp = True,
+    tag = "latest",
+)
+
+filegroup(
+    name = "package-srcs",
+    srcs = glob(["**"]),
+    tags = ["automanaged"],
+    visibility = ["//visibility:private"],
+)
+
+filegroup(
+    name = "all-srcs",
+    srcs = [":package-srcs"],
+    tags = ["automanaged"],
+    visibility = ["//visibility:public"],
+)

--- a/test/kubemark/start-kubemark.sh
+++ b/test/kubemark/start-kubemark.sh
@@ -214,6 +214,25 @@ function create-and-upload-hollow-node-image {
   echo "Created and uploaded the kubemark hollow-node image to docker registry."
 }
 
+# Use bazel rule to create a docker image for hollow-node and upload
+# it to the appropriate docker container registry for the cloud provider.
+function create-and-upload-hollow-node-image-bazel {
+  RETRIES=3
+  for attempt in $(seq 1 ${RETRIES}); do
+    if ! bazel run //cluster/images/kubemark:push --define PROJECT="${PROJECT}"; then
+      if [[ $((attempt)) -eq "${RETRIES}" ]]; then
+        echo "${color_red}Image push failed. Exiting.${color_norm}"
+        exit 1
+      fi
+      echo -e "${color_yellow}Make attempt $(($attempt)) failed. Retrying.${color_norm}" >& 2
+      sleep $(($attempt * 5))
+    else
+      break
+    fi
+  done
+  echo "Created and uploaded the kubemark hollow-node image to docker registry."
+}
+
 # Generate secret and configMap for the hollow-node pods to work, prepare
 # manifests of the hollow-node and heapster replication controllers from
 # templates, and finally create these resources through kubectl.
@@ -439,7 +458,11 @@ start-master-components
 # Setup for hollow-nodes.
 echo ""
 echo -e "${color_yellow}STARTING SETUP FOR HOLLOW-NODES${color_norm}"
-create-and-upload-hollow-node-image
+if [[ "${KUBEMARK_BAZEL_BUILD:-}" =~ ^[yY]$ ]]; then
+  create-and-upload-hollow-node-image-bazel
+else
+  create-and-upload-hollow-node-image
+fi
 create-kube-hollow-node-resources
 wait-for-hollow-nodes-to-run-or-timeout
 

